src/command_modules/azure-cli-network/HISTORY.rst

```diff
@@ -14,6 +14,7 @@ Release History
                      Fix issue where quoted TXT records were incorrectly exported without escaped quotes.
 * `dns zone import`: Fix issue where certain records were imported twice.
 * Restored `vnet-gateway root-cert` and `vnet-gateway revoked-cert` commands.
+* `network public-ip create`: Fix `--tags` option.
```

Comment:
This has already been released. You need to add a new section and move your fix into there. Also, update the version in setup.py. That's why you have a CI failure.

Reply:
Thanks! I will update it later today.